app/test/unit/git/for-each-ref-test.ts

```diff
@@ -71,26 +71,21 @@ describe('git/for-each-ref', () => {
     })
 
     it('filters branches differing from upstream using for-each-ref', async () => {
-      const allBranches = await getBranches(repository)
+      const branches = await getBranchesDifferingFromUpstream(repository)
 
-      const branches = await getBranchesDifferingFromUpstream(
-        repository,
-        allBranches
-      )
-
-      const branchNames = branches.map(branch => branch.name)
+      const branchNames = branches.map(branch => branch.ref)
```

Comment:
Nit: Suggest to rename this variable to branchRefs

Reply:
Nice catch!